langchain4j-core/src/main/java/dev/langchain4j/internal/JsonParsingException.java

```diff
@@ -0,0 +1,10 @@
+package dev.langchain4j.internal;
```

Comment:
```suggestion
package dev.langchain4j.exception;
```
Or is it meant to be used only internally?